plugin.program.autowidget/resources/language/resource.language.en_gb/strings.po

```diff
@@ -726,42 +726,68 @@ msgctxt "#30132"
 msgid "Country Icons"
 msgstr "Country Icons"
 
-#: /resources/lib/menu.py:30
+#: /resources/lib/menu.py:31
 msgctxt "#30133"
 msgid "Next"
 msgstr "Next"
 
-#: /resources/lib/menu.py:31
+#: /resources/lib/menu.py:32
 msgctxt "#30134"
 msgid "Next Page"
 msgstr "Next Page"
 
-#: /resources/lib/menu.py:32
+#: /resources/lib/menu.py:33
 msgctxt "#30135"
 msgid "Previous"
 msgstr "Previous"
 
-#: /resources/lib/menu.py:33
+#: /resources/lib/menu.py:34
 msgctxt "#30136"
 msgid "Previous Page"
 msgstr "Previous Page"
 
-#: /resources/lib/menu.py:34
+#: /resources/lib/menu.py:35
 msgctxt "#30137"
 msgid "Back"
 msgstr "Back"
 
-#: /resources/lib/menu.py:35
+#: /resources/lib/menu.py:36
 msgctxt "#30138"
 msgid "Page"
 msgstr "Page"
 
-#: /resources/lib/menu.py:740
+#: /resources/lib/menu.py:753 /resources/lib/refresh.py:332
+#: /resources/lib/common/cache.py:276
 msgctxt "#30139"
 msgid "Error showing {}"
 msgstr "Error showing {}"
 
-#: /resources/lib/menu.py:750
+#: /resources/lib/menu.py:764 /resources/lib/refresh.py:343
 msgctxt "#30140"
 msgid "No content found for {}"
 msgstr "No content found for {}"
+
+#: /resources/lib/refresh.py:95
+msgctxt "#30141"

```

Comment:
I "viewed" all of the other `.po` files, but these are the new translations I added.